data/data/azure/bootstrap/main.tf

```diff
@@ -18,8 +18,9 @@ provider "azurerm" {
 }
 
 data "azurerm_storage_account" "storage_account" {
-  name                = var.storage_account_name
-  resource_group_name = var.resource_group_name
+  name                             = var.storage_account_name
+  resource_group_name              = var.resource_group_name
+  cross_tenant_replication_enabled = false
```

Comment:
```suggestion
```
`cross_tenant_replication_enabled` does not seem to be an attribute on the storage account data source. We use the data source when using an existing resource, so it probably needs to be set at creation time. Just remove this change.

Reply:
@patrickdillon This should be the exact variable that we need. I see that we have a version of 3.43.0 in terraform/providers/azurerm. Here is the link to that resource and version (https://registry.terraform.io/providers/hashicorp/azurerm/3.43.0/docs/resources/storage_account). I can see the variable listed here and the bug itself says that we need to set this to false as it currently defaults to False if we use the latest versions of the provider.